src/scripts/site.js

```diff
@@ -420,586 +644,747 @@ var Dashboard = {
-            "bower_components/emby-webcomponents/htmlaudioplayer/plugin",
-            "bower_components/emby-webcomponents/htmlvideoplayer/plugin",
-            "bower_components/emby-webcomponents/photoplayer/plugin",
-            "bower_components/emby-webcomponents/youtubeplayer/plugin"
-        ];
+        var list = ["bower_components/emby-webcomponents/playback/playbackvalidation", "bower_components/emby-webcomponents/playback/playaccessvalidation", "bower_components/emby-webcomponents/playback/experimentalwarnings", "bower_components/emby-webcomponents/htmlaudioplayer/plugin", "bower_components/emby-webcomponents/htmlvideoplayer/plugin", "bower_components/emby-webcomponents/photoplayer/plugin", "bower_components/emby-webcomponents/youtubeplayer/plugin"];
+
         if ("cordova" === self.appMode) {
             list.push("cordova/chromecast");
         }
+
         if ("android" === self.appMode) {
             list.push("cordova/externalplayer");
         }
+
         if (appHost.supports("remotecontrol")) {
             list.push("bower_components/emby-webcomponents/sessionplayer");
+
             if (browser.chrome || browser.opera) {
                 list.push("bower_components/emby-webcomponents/chromecast/chromecastplayer");
             }
         }
-        for (var i = 0, length = externalPlugins.length; i < length; i++) list.push(externalPlugins[i]);
-        return new Promise(function(resolve, reject) {
-            Promise.all(list.map(loadPlugin)).then(function() {
-                require(["packageManager"], function(packageManager) {
-                    packageManager.init().then(resolve, reject)
-                })
-            }, reject)
-        })
+
+        for (var i__y = 0, length = externalPlugins.length; i__y < length; i__y++) {
+            list.push(externalPlugins[i__y]);
+        }
+
+        return new Promise(function (resolve, reject) {
+            Promise.all(list.map(loadPlugin)).then(function () {
+                require(["packageManager"], function (packageManager) {
+                    packageManager.init().then(resolve, reject);
+                });
+            }, reject);
+        });
     }
 
     function loadPlugin(url) {
-        return new Promise(function(resolve, reject) {
-            require(["pluginManager"], function(pluginManager) {
-                pluginManager.loadPlugin(url).then(resolve, reject)
-            })
-        })
+        return new Promise(function (resolve, reject) {
+            require(["pluginManager"], function (pluginManager) {
+                pluginManager.loadPlugin(url).then(resolve, reject);
+            });
+        });
     }
 
     function enableNativeGamepadKeyMapping() {
-        return !(!window.navigator || "string" != typeof window.navigator.gamepadInputEmulation) && (window.navigator.gamepadInputEmulation = "keyboard", !0)
+        if (!(!window.navigator || "string" != typeof window.navigator.gamepadInputEmulation)) {
```

Comment:
I hope I remember De Morgan's laws correctly...
```suggestion
        if (window.navigator && "string" == typeof window.navigator.gamepadInputEmulation) {
```